views/widget.html

```diff
@@ -2,7 +2,15 @@
 {% set id = apos.utils.generateId() %}
 {% set widget = data.widget %}
 {% set classPrefix = data.widget.classPrefix %}
-<script src="https://maps.google.com/maps/api/js?key={{ widget.googleApiKey }}&libraries=places" type="text/javascript"></script>
+<script src="https://maps.google.com/maps/api/js?key={{ widget.googleApiKey }}&libraries=places
```

Comment:
We should include `defer` on this. Or use `apos.templates.append('body', ...)` to put it at the end of the DOM.

Reply:
Nice for `defer`. Should I add `async` too?

For `apos.templates.append` where would you use it? I tried it but did not understand how to get widget values.